.github/workflows/junit4.yml

```diff
@@ -20,15 +20,8 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
-    - name: Run tests
-      continue-on-error: true
+    - name: Run junit4 tests
       env:
         SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
         SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
-      run: mvn test -pl best-practice -Dtest=DesktopTests
-    - name: Retry failed tests
-      continue-on-error: false
-      id: retry1
-      if: steps.test.outcome=='failure'         # check the step outcome, retry 1st time
-      run: |
-        mvn test -pl best-practice -Dtest=DesktopTests
+      run: mvn test -pl selenium-junit4-examples -X
```

Comment:
@titusfortner the reason why I was limiting many of these to say `DesktopTests` was because of the instability of Sauce environment. I found issues with VMs and Devices not being available or connection timeouts. As a result I limited my variables in CI to avoid false positives and have a trustworthy pipeline.

Reply:
> did you see the 3 failing build checks?

The ones I saw were visual and I can't check them because it's your screener. Are there additional ones?

> not sure if it changed but a while ago we had a conversation with the Docs team that we should just change our repo and they will deal with the broken links after...

Yeah, it's good to know they don't want to slow us down, but I want to make sure we have a good system in place to avoid shooting ourselves.

As for not running everything... I have concerns. Rather than having yaml files determine what gets run, let's create tags for the things that we consider unstable and filter them out via maven directly.